config/routes.rb

```diff
@@ -1,4 +1,9 @@
-Rails.application.routes.draw do
+  root 'users#index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
+
+
```

Comment:
### OPTIONAL
- [ ] It would be better if we capitalize all our commit messages, this makes our repo looks professional and higlights attention to detail